README.md

```diff
@@ -9,17 +9,32 @@ There is no "release" yet. The current milestone is "1.5 Stable", which is meant
-See your platform below:
+1. Make sure that you have a copy of Descent 3. You can purchase a copy of Descent 3 from [GOG](https://www.gog.com/game/descent_3_expansion) or [Steam](https://store.steampowered.com/app/273590/Descent_3/).
+2. Install Descent 3.
 
-#### Windows
-If still using the Descent 3 launcher, copy your `Descent3.exe` binary to your install folder and rename it `main.exe` (back up your old one). Otherwise, drop in `Descent3.exe` and play!
+    - **Note for Steam users:** If you own Descent 3 on Steam, then it’s recommended that you install the Windows version of the game even if you’re running macOS or Linux. [If you do not do this, then cutscenes will not work.](https://github.com/DescentDevelopers/Descent3/issues/240) You can use either [Steam Play](https://help.steampowered.com/en/faqs/view/08F7-5D56-9654-39AF) or [SteamCMD](https://developer.valvesoftware.com/wiki/SteamCMD#Cross-Platform_Installation) to install the Windows version of the game on macOS or Linux.
 
-#### MacOS
-Right-click Descent3.app, click Show Package Contents. Back up your `Descent3` binary and drop your built `Descent3` binary into the install (Contents/MacOS) folder.
+    - **Note for non-Windows users:** If you have the Windows version of the game on CDs but you don’t want to use Windows to install them, then check out [this guide](https://github.com/DescentDevelopers/Descent3/discussions/273#discussioncomment-9300692).
 
-#### Linux
-Back up your `Descent3` binary and drop your built `Descent3` binary into the install folder.
+3. If your version of Descent 3 is older than v1.4, then [update it to v1.4](http://descent3.com/downloads.php).
+4. Create a new folder named `D3-open-source`.
+5. Copy the following files from your installation of Descent 3 to `D3-open-source`:
+    - All `.hog` files, except for `d3-linux.hog`
+    - The `missions` folder
+    - _(Optional)_ All `.pld` files
+    - _(Optional)_ The `demo` folder
+    - _(Optional)_ The `movies` folder
+6. Create the following folders in `D3-open-source`:
+    - `cache/`
+    - `custom/`
+    - `custom/cache/`
+7. Obtain new Descent 3 engine files:
+    - If you want to use pre-built binaries, then download one of the artifacts from our latest CI run. You can find a list of CI runs [here](https://github.com/DescentDevelopers/Descent3/actions/workflows/build.yml?query=branch%3Amain).
+    - If you want to build the engine files yourself, the follow [these instructions](#building). Once you build the engine files, they’ll be put in `builds/<platform>/Descent3/<build-type>/`. For example, if you’re using Linux and you create a “Release” build, then the files will be located at `builds/linux/Descent3/Release`.
+8. Copy all of the new engine files into `D3-open-source`.
+9. Run the game:
+    - On Windows, run `D3-open-source\Descent3.exe`.
```

Comment:
Windows line should be under the Windows section.

Reply:
This PR removes the Windows, MacOS and Linux sections that were in the Usage section. Do you want me to add them back?

Reply:
Nah, that's fine. This will probably be changed often since we're adding more data to the artifacts like .d3cs and .d3ms. Pretty soon it'll just be a simple "copy these game assets into the artifact directory".